app/javascript/components/search/genes/StudyGeneExpressions.jsx

```diff
@@ -17,7 +18,7 @@ import { getDefaultClusterParams, getAnnotationValues, emptyDataParams } from '~
   */
 export default function StudyGeneExpressions({ study }) {
   const [clusterParams, setClusterParams] = useState(_clone(emptyDataParams))
-  const [annotationList, setAnnotationList] = useState(null)
+  const [annotationList, setAnnotationList] = useState(study.annotation_list ?? null)
   let controlClusterParams = _clone(clusterParams)
 
```

Comment:
Add a `getAnnotationFromIdentifier` helper method to cluster-utils.js that mirrors the existing `getIdentifierFromAnnotation`, and then you can do
```suggestion
const defaultAnnotation = getAnnotationFromIdentifier(study.default_annotation)
const defaultAnnotationIsNumeric = defaultAnnotation?.type === 'numeric'
```
For bonus points, update line 80 of ExploreTabRouter.jsx to use the new helper function